client/src/modules/multiple_payroll/templates/action.tmpl.html

```diff
@@ -20,7 +20,7 @@
     <li class="divider"></li>
 
     <li>
-      <a data-method="employee" ui-sref="employeeRegistry({ filters : [{ key : 'hrreference', value : row.entity.hrreference, cacheable:false }]})" href>
```

Comment:
Hmm... I think I wrote this code and I think I wrote it wrong.  Sorry!  I think it should be:
```html
<a 
  data-method="employee"
  ui-sref="employeeRegistry({ filters : [{ key : 'uuid', value : row.entity.employee_uuid, displayValue: row.entity.hrreference, cacheable:false }]})"
  href> 
```